Makefile

```diff
@@ -0,0 +1,13 @@
+image_build:
+	sudo docker build -t test_tron .
```

Comment:
In general, I avoid sudo. The better practice is to add the user to the docker group so sudo is not necessary. Please see http://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo